src/vmm_mad/remotes/lib/lxd/opennebula_vm.rb

```diff
@@ -240,9 +242,13 @@ def disk_mountpoint(disk_id)
     def disk_source(disk)
         disk_id = disk['DISK_ID']
 
-        if disk['TYPE'] == 'RBD'
+        if disk['DISK_TYPE'] == 'RBD'
```

Comment:
Why using DISK_TYPE, what are the differences between TYPE and DISK_TYPE?

Reply:
if the disk is volatile, then `'TYPE'` is `'fs'`. It shouldn't affect this case using 'TYPE' or 'DISK_TYPE', but the `new_disk_mapper` method was sensitive to this, so disk source was changed accordingly.